app/controllers/v2/application_controller.rb

```diff
@@ -102,7 +102,7 @@ def select_fields(fields)
           if association
             "#{association}.#{column} AS #{association}__#{column}"
           else
-            [resource.table_name, column].join('.')
+            column.is_a?(ApplicationRecord::AN::Node) ? column.to_sql : [resource.table_name, column].join('.')
```

Comment:
This is not the final solution, but I needed something for interpolating arel columns, such as `V2::System::OS_MAJOR_VERSION` and `V2::System::OS_MINOR_VERSION`. After the aggregation is done, this will look much nicer 😉 

![i1-will-refactor-that-code-later-today-and-other-31878628](https://github.com/RedHatInsights/compliance-backend/assets/649130/68673719-5ceb-4bb3-8fa8-50034cffc142)